phpunit.xml

```diff
@@ -1,6 +1,6 @@
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/7.5/phpunit.xsd"
```

Comment:
Why?

Reply:
Lowest supported PHPUnit version is 7.5, use vendor version will break config validation for higher PHPUnit version.